framework/src/modules/chain/index.js

```diff
@@ -126,8 +126,9 @@ module.exports = class ChainModule extends BaseModule {
 				handler: async action => this.chain.actions.blocks(action),
 				isPublic: true,
 			},
-			blocksCommon: {
-				handler: async action => this.chain.actions.blocksCommon(action),
+			getHighestCommonBlockId: {
```

Comment:
Security wise, this action needs to be public but this also implies someone can write a custom script that tries to overload the node by requesting to find a common ID by sending continuously fake IDs. We should take care of a rate-limiting security measure of some sort.